geomesa-accumulo/geomesa-accumulo-datastore/pom.xml

```diff
@@ -144,6 +145,12 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kerby</groupId>
+            <artifactId>kerb-simplekdc</artifactId>
+            <version>1.0.0-RC2</version>
```

Comment:
I've actually removed the dep, since although I went to the trouble of creating a mini KDC server, it wasn't actually required anywhere! (legacy from some previous code trying to get TGTs working)